Packs/ProofpointThreatResponse/README.md

```diff
@@ -1,3 +1,55 @@
 # Proofpoint Threat Response
+<~XSIAM>
+## Overview
+Proofpoint Threat Response is a security solution that helps organizations detect, analyze, and respond to threats quickly. It automates threat investigation and incident response to reduce risk and response time. By integrating with other security tools, it streamlines workflows and improves overall security operations.
 
-This pack includes Cortex XSIAM content.
+## This pack includes:
+- Rest API integration for security incidents
+- Modeling Rules
+
+## Configure Proofpoint Threat Response Event Collector in Cortex XSIAM
+
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| Server URL (e.g., https://192.168.0.1) |  | True |
+| API Key for the authentication. |  | True |
```

Comment:
the parameter names and descriptions are combined here in the first column, they need to be in separate columns. maybe something happened when generating the README from the YAML file?